sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/VectorLayerVisualizer.cs

```diff
@@ -50,12 +51,20 @@ public override string Key
 		[SerializeField]
 		[NodeEditorElementAttribute("Custom Stacks")]
 		public List<TypeVisualizerTuple> Stacks;
-		
-		private GameObject _container;
+
+		[NonSerialized]
+		private Dictionary<UnityTile, List<int>> _activeCoroutines;
+
+		[NonSerialized]
+		private int _entityPerCoroutine = 20;
```

Comment:
I think I was changing that param back&forth while testing and I generally put that nonserialized thing to private field these days as I had some crazy bugs because of that.

Reply:
we can remove&public that when we finalize this by the way